java/tomcat-native/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem 1.0
 
 name                tomcat-native
-version             1.2.14
+version             1.2.16
 categories          java www
 maintainers         thebishops.org:matt
```

Comment:
This is the maintainers line. So in your case, it should be:
`maintainers	{thebishops.org:matt @mattbishop} openmaintainer`

Reply:
IOC